ingestion_service/producer/core/routers/api_endpoints_input.py

```diff
@@ -279,9 +306,14 @@ async def ingest_knowledge_graphs_batch(
         user: Annotated[LoginUserIn, Depends(get_current_user)],
         files: List[UploadFile] = File(...),
         posting_user: str = Form(...),
+        graph: str = Form(...)
+
 ):
     if not files:
         raise HTTPException(status_code=400, detail="No files provided")
+    named_graph_iri = named_graph_exists(graph)
+    if named_graph_iri["status"] != True:
```

Comment:
What does this mean if the status is not `True`? Should the HTTP response back not be a 200?